src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1605,4 +1597,39 @@ Please specify one of them using the --source option to proceed.</value>
     <value>Select installed package scope filter (user or machine)</value>
     <comment>This argument allows the user to select installed packages for just the user or for the entire machine.</comment>
   </data>
-</root>
+  <data name="PinAdded" xml:space="preserve">
+    <value>Pin added successfully</value>
+  </data>
+  <data name="PinAlreadyExists" xml:space="preserve">
+    <value>The pin already exists</value>
+  </data>
+  <data name="PinExistsOverwriting" xml:space="preserve">
+    <value>A pin already exists for this package. Overwriting due to the --force argument.</value>
+    <comment>{Locked="--force"}</comment>
+  </data>
+  <data name="PinExistsUseForceArg" xml:space="preserve">
+    <value>A pin already exists for this package. Use the --force argument to overwrite it.</value>

```

Comment:
Is there a reason not to use the `{0}` and the package name here? Just curious, since the `PinDoesNotExist` message does use it

Reply:
No reason. I'll change that. It's part of what I meant with "Make this pretty. Lots of places could use better or more messages." in the PR description :)